### DISCUSSION TITLE
is there a GUI tool to manage milvus collections? Can I use Attu without kubernetes?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18191] [Q&A and General discussion] [answered]
is there a GUI tool to manage milvus collections? Can I use Attu without kubernetes?

**Comment by martinenkoEduard** [comment, marked as answer]
Found it! https://github.com/zilliztech/attu

**Reply by xiaofan-luan** [reply]
Yep! we are still working on attu, so feel free to give us more comment
